config/locales/en.yml

```diff
@@ -152,8 +152,9 @@ en:
       access_throttled: You've attempted to sign up too many times and your access had been throttled. Try again later.
     create_auth_token:
       unfound_alert_html: |
-        Couldn't find a publisher with that email address. Please try again or
-        <a href="%{link}">sign up for an account</a>.
+        Couldn't find a publisher with that email address. Please try again, or
+        you may
+        <a data-method="post" href="%{create_publisher_path}">create an account with the email %{email}</a>.
```

Comment:
minor: this doesn't work without javascript and falls back to a 404 at `[GET] "/publishers"`

is there a way to make it work?